audiomentations/augmentations/transforms.py

```diff
@@ -1241,6 +1241,56 @@ def apply(self, samples, sample_rate):
         return distorted_samples
 
     
+class InterruptPulse(BaseWaveformTransform):
+    """
+    Add interrupt pulses to the input audio.
```

Comment:
We should mention here that this transform could be useful for ECG augmentation. Further, we can use some of the description from @darkcurrent:

> The main purpose is simulate the signals when the ECG electrodes are in lead-off condition. It can also be used for audio data augmentation.
The function generates random number of flat pulses with some noise on it. The pulses' amplitudes are between signal max-min range and pulses have random width.

Reply:
Added.